support-frontend/stories/screens/supporterPlusThankYou/AusSupporterPlusThankYou.stories.tsx

```diff
@@ -262,7 +260,6 @@ RecurringNotSignedIn.decorators = [
 		store.dispatch(setStorybookUser(true));
 		store.dispatch(setCountryInternationalisation('AU'));
 		store.dispatch(setProductType(contributionType));
-		store.dispatch(setUserTypeFromIdentityResponse('guest'));
```

Comment:
I have kind of broken the point of these - but this page "supporterplusthankyou" component is being replaced soon